entrypoint.sh

```diff
@@ -2,8 +2,10 @@
 
 set -e
 
-if [ ! -f "/cert.pem" ]; then
-    echo "Certificate file '/cert.pem' not found. Make sure it is mounted as a volume when starting the container."
+export CERT_PATH=${CERT_PATH:-"/cert.pem"}
+
+if[ ! -f ${CERT_PATH} ]; then
```

Comment:
I think there is a space missing here after the `if`. Also, please enclose the variable in quotes. Otherwise there will be a syntax error in case the variable is not set. With quotes it will resolve to an empty string if the variable is not set (`if [ ! -f "${CERT_PATH}" ]; ...`).